### PR TITLE
Create data model README file

### PR DESCRIPTION
**Pull request by sharifX**
Create a simple data model README file to describe the folders.

### CODE REVIEW

File: data-model/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+This section includes the data model for differnet FAIR Digital Objects.
```
Comment (wouteraddink) [inline data-model/README.md:2]: different

**Review by wouteraddink — CHANGES_REQUESTED** [review]
just a few typos to fix

File: data-model/README.md (hunk 1 above)
Comment (sharifX) [reply data-model/README.md]: fixed

File: data-model/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+This section includes the data model for different FAIR Digital Objects. 
+
+## Directory Structure
+
+- **annotations/**: Contains schemas and definitions related to annotation data model.
```
Comment (samleeflang) [inline data-model/README.md:6]: I think this folder should no longer be there. They have become part of the fdo-type

File: data-model/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Data Modeling Approach
+
+Our approach to data modeling involves:
+
+1. **Reviewing Existing Standards**: We review existing terms and standards such as Darwin Core and ABCD to identify relevant concepts and definitions.
+2. **Harmonisation**: We harmonise these existing terms to create a specification that meets the needs of digital specimens lifecycle in the DiSSCo infrastructure.
+3. **Specification Development**: Based on the harmonised terms, we develop detailed specifications and schemas that define the structure and content of digital specimen and related data.
+
```
Comment (samleeflang) [inline data-model/README.md:29]: Could mention something about versioning, that each object is individually versioned. Oh and maybe point to schemas.dissco.tech?

**Review by samleeflang — COMMENTED** [review]
Small comment

File: data-model/README.md (hunk 2 above)
Comment (sharifX) [reply data-model/README.md]: fixed

File: data-model/README.md (hunk 3 above)
Comment (sharifX) [reply data-model/README.md:28]: added.

File: data-model/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- **nginx/**: Configuration files for the Nginx server used in the deployment of the data model services.
+- **Dockerfile**: A Dockerfile for building and deploying the data model services in a containerized environment.
+
+
+## Data Modeling Approach
+
+Our approach to data modeling involves:
+
+1. **Reviewing Existing Standards**: We review existing terms and standards such as Darwin Core and ABCD to identify relevant concepts and definitions.
+2. **Harmonisation**: We harmonise these existing terms to create a pecification that meets the specific needs of digital specimens lifecycle in the DiSSCo infrastructure.
```
[the remark this answers is not in the document]
Comment (sharifX) [reply data-model/README.md]: fixed

**Review by samleeflang — APPROVED** [review]
:ice_cream:

**Pull request merged**